pkgs/development/python-modules/numpy/numpy-distutils-C++.patch

```diff
@@ -26,6 +25,6 @@ index 6ed5eec..82a88b5 100644
 +            self.spawn(self.compiler_so + cc_args + [src, '-o', obj] + deps +
 +                       extra_postargs, display = display)
 +
-     except DistutilsExecError:
-         msg = str(get_exception())
+     except DistutilsExecError as e:
```

Comment:
hope they don't run pylint as part of the tests because pylint doesn't like variables this short ;)